README.md

```diff
@@ -86,24 +93,33 @@ sudo apt install inkscape optipng
 
 2. Clone the repository.
 
-```
+```sh
 git clone https://github.com/pop-os/gtk-theme.git
 cd gtk-theme
 ```
 
 3. Generate the theme files.
 
-```
+```sh
 meson build && cd build
 ninja
 ```
 
 4. Install the theme.
 
-```
+```sh
 ninja install
 ```
 
+Or if you prefer to install it locally:
```

Comment:
Definitely a huge improvement here! :tada:

Reply:
Should the two sets of instructions be consistent between `cd`'ing into the `build` directory vs. using the `-C "build"` option?

Reply:
https://ninja-build.org/manual.html#_running_ninja

Effectively change into the build directory and run `install`.

You could checkout the branch and run it and see if you're happy with it.

Reply:
@mbana I know they are equivalent, I'm saying it would be more consistent to use one method for both examples.

Reply:
Decision is up to you. Let me know what you decide and I'll make the change otherwise, I'll leave it as-is.

Reply:
@isantop @jacobgkau, I'll close the PR, if you're not happy with it. Please just let me know.

Reply:
I attempted to address this myself, but I do not have permissions to commit to this branch.

@mbana Can we please make changes so it looks like this:

~~~
1. If previous versions were installed/existed, remove them first:

 ```sh
 sudo apt remove pop-gtk-theme
 sudo rm -rf /usr/share/themes/Pop*
 rm -rf ~/.local/share/themes/Pop*
 rm -rf ~/.themes/Pop*
```

2. Clone the repository:

```sh
git clone https://github.com/pop-os/gtk-theme.git
cd gtk-theme
```

3. Generate the theme files and install the theme:

```sh
meson build
ninja -C "build" install
```

Or, if you prefer to install it locally:

```sh
meson -Dprefix=$HOME/.local build
ninja -C "build" install
```
~~~

Assuming @isantop is okay with this, I think it makes more sense for step-by-step installation instructions. I realize knowing how to generate the files separately from installing may be valuable in certain circumstances, but that may be outside the scope of this project's readme file.

Reply:
OK. Please see the latest commit.